src/types/types.ts

```diff
@@ -522,8 +588,23 @@ export const enum ErrorStatusCode {
   /** @see https://docs.meilisearch.com/errors/#api_key_not_found */
   API_KEY_NOT_FOUND = 'api_key_not_found',
 
-  /** @see https://docs.meilisearch.com/errors/#missing_parameter */
-  MISSING_PARAMETER = 'missing_parameter',
+  /** @see https://docs.meilisearch.com/errors/#immutable_api_key_uid */
+  IMMUTABLE_API_KEY_UID = 'immutable_api_key_uid',
+
+  /** @see https://docs.meilisearch.com/errors/#immutable_api_key_actions */
+  IMMUTABLE_API_KEY_ACTIONS = 'immutable_api_key_actions',
+
+  /** @see https://docs.meilisearch.com/errors/#immutable_api_key_indexes */
+  IMMUTABLE_API_KEY_INDEXES = 'immutable_api_key_indexes',
+
+  /** @see https://docs.meilisearch.com/errors/#immutable_api_key_expires_at */
+  IMMUTABLE_API_KEY_EXPIRES_AT = 'immutable_api_key_expires_at',
+
+  /** @see https://docs.meilisearch.com/errors/#immutable_api_key_created_at */
+  IMMUTABLE_API_KEY_CREATED_AT = 'immutable_api_key_created_at',
+
+  /** @see https://docs.meilisearch.com/errors/#immutable_api_key_update_at */
+  IMMUTABLE_API_KEY_UPDATE_AT = 'immutable_api_key_update_at',
```

Comment:
Since we have `IMMUTABLE_API_KEY_CREATED_AT` above, why isn't this one called `IMMUTABLE_API_KEY_UPDATED_AT` ("updated" with a "d")?